jest.config.base.js

```diff
@@ -17,6 +17,11 @@ module.exports = (packageName) => ({
   snapshotSerializers: ['enzyme-to-json/serializer'],
   moduleDirectories: ['node_modules'],
   collectCoverageFrom: ['<rootDir>/**/*.{js,jsx,ts,tsx}'],
+  moduleNameMapper: {
+    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
+      '<rootDir>/../../script/jest/fileMock',
+    '\\.(css|scss)$': '<rootDir>/../../script/jest/styleMock',
+  },
```

Comment:
This allows us to use mocks for css instead of the actual assets; copied this behavior from gamut